### DISCUSSION TITLE
Change the chart type

### DISCUSSION THREAD

**Discussion by HarshaKosala** [tradingview/lightweight-charts#789] [Q&A] [answered]
Hi,
Is there a way to change the chart type from area to candle without again set data to the candle series. Just change the chart type for the same data set, same as we do in the advance chart.

Thanks.

**Comment by timocov** [comment, marked as answer]
No, it's impossible. The reason why it is so because different series types have different type of data and options, thus you cannot just "convert" it without providing the rest of the data and new options.

**Comment by qrq** [comment]
Yes you can , just get at first data with all [ time , open,high,low,close ] this is work with candlestick , if you want to plot same data with area chart all you need time and close from same set and no need to import it again
